Crash 1seconds

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Game Version

4.21.0

### Describe the bug

**Platform:** Android
**Version:** 4.21.0 (Номер сборки: 1235)
**Rulesets:** [Civ V - Vanilla, Civ V - Gods & Kings]
**Last Screen:** `com.unciv.ui.crashhandling.CrashScreen`
**A-star pathing enabled:** false

--------------------------------

Device Model: RMX3370
API Level: 36
System Memory: 7479 MB
    Available (used by Kernel): 3522 MB
    System Low Memory state: false
    Java heap limit: 512 MB
    Java heap free: 66 MB

--------------------------------


**Message:**
```
com.badlogic.gdx.utils.GdxRuntimeException: Error writing file: SaveFiles/Autosave (External)
	at com.badlogic.gdx.files.FileHandle.writeString(FileHandle.java:386)
	at com.unciv.logic.files.UncivFiles.saveGame(UncivFiles.kt:180)
	at com.unciv.logic.files.UncivFiles.saveGame(UncivFiles.kt:169)
	at com.unciv.logic.files.UncivFiles.saveGame$default(UncivFiles.kt:167)
	at com.unciv.logic.files.Autosaves.autoSave(UncivFiles.kt:507)
	at com.unciv.logic.files.Autosaves$requestAutoSaveUnCloned$job$1.invokeSuspend(UncivFiles.kt:496)
	at com.unciv.logic.files.Autosaves$requestAutoSaveUnCloned$job$1.invoke(Unknown Source:8)
	at com.unciv.logic.files.Autosaves$requestAutoSaveUnCloned$job$1.invoke(Unknown Source:4)
	at com.unciv.utils.ConcurrencyKt$launchCrashHandling$1.invokeSuspend(Concurrency.kt:106)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:207)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:207)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt.wrapCrashHandling$lambda$0(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt.$r8$lambda$ZmD3ejHdHQgJjNvghy9393ssfKE(Unknown Source:0)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$$ExternalSyntheticLambda0.invoke(D8$$SyntheticClass:0)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt.wrapCrashHandlingUnit$lambda$0(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt.$r8$lambda$4MlUfb0a_98IfrUPubAef8aLYuM(Unknown Source:0)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$$ExternalSyntheticLambda1.invoke(D8$$SyntheticClass:0)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher.dispatch$lambda$0(Concurrency.kt:207)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher.$r8$lambda$L3K1aau81hhVdGYYei_E7yWI2wU(Unknown Source:0)
	at com.unciv.utils.Dispatchers$CrashHandlingDispatcher$$ExternalSyntheticLambda0.run(D8$$SyntheticClass:0)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1154)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:652)
	at java.lang.Thread.run(Thread.java:1563)
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Error writing file: SaveFiles/Autosave (External)
	at com.badlogic.gdx.files.FileHandle.writer(FileHandle.java:363)
	at com.badlogic.gdx.files.FileHandle.writeString(FileHandle.java:383)
	... 24 more
Caused by: java.io.FileNotFoundException: /storage/emulated/0/Android/data/com.unciv.app/files/SaveFiles/Autosave: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:574)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:259)
	at com.badlogic.gdx.files.FileHandle.writer(FileHandle.java:355)
	... 25 more
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:574)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:274)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:574)
	at android.app.AndroidForwardingOs.open(AndroidForwardingOs.java:120)
	at libcore.io.IoBridge.open(IoBridge.java:560)
	... 27 more

```
**Save Mods:**
```
[Civ V - Gods & Kings]
```

**Save Data:**
<details><summary>Show Saved Game</summary>

```
H4sIAAAAAAAA/+2da4/cNrKG/8qgP50DWIHuF3+zJ5s4izj2sZ0Y2IU/aLrVM4J7WrN9sTMb+L8fSi1Kag2L4ktSY+8myGKTUUsUJVFvPSoWq/5YXOU79r8y3+4XT/9YFNtlfnt3W2wP7M9//rFYVsftYVV93i6epk8Wd9W+PJTVtt7z98XT8MnifvHU8fwvTxb7u/zztlj9ui3rI90vTyaO1TooaE6odahzOtaJdI6NTucNdI71vNPBgmM/sG3L8lO5Kf+d14e0t7z8dMn++7A7LtuNf5z2++n7xdPF8/5xNRt/yW+L8eZif8cOzK+Ll3n9/7tTC8Xvd5tqV6zeFNen7rFt19VmVWyfXRfNw98f6t9f5Hd35bbY79s7fb0r8sProrrbFPVOd1XJBscP1e6X4vfDj/VvPxbbYpdvLuvb0Jxs0Wy+aLcvnvqu++VLfWc25bIs2gv617HYH846uCs2Zd21843Hcrs/23Ioljf1Sep/798U+yLfLW+KFbt3i2fXu3J53ByOu2JR39xP5ZJd0v3g8OZ5SW/w3/O7fHt2b/kWxdu6uKoOh+r2TXl9c+gGQdKPvkN193Ox7n/ym5+yL4/9OLbVoVyXy2bkNTs/O7w95LvDu+OOXYbLntcmvy927+7v6pvw4njb3ITzh3i3q34vb8vDfX2a58Xmepev6r1/qbYF2/nZkd3lMh9s2OVXw79fV5t7dm3DTZf5XXHxrnmb+Kbf8s2mOBwGO7Frqc6Ouak+Dhv5x5ENhBv+t8lo2x/yw/5FWT+L5iI91ujb8Bf30n3t/ui+c3/w0xee/9575i7sjM1Dc3MbHTjcj8fp4upYbg7P2f+tyu11vd/idb7Jl6xJNmIHO//fsTgW9c8vq+2xlvN6h3L7elex8bPfP2i32+1pWI8N1tS2WB6K1buKPY/ykG/Y0Dgc2a7sqGLwKnTvAFOedvi3F/m2Gb0n4Vuz4VV3+LL8xG5f/9BZl9geiygIosjPMid1V64Txp7vZK6XOkXmJ+t8vQrcyKv33b/asUe2ZWP71Cfem011GsUjqW9el+3pHX5RbavNcXNsRvDdcdPtv66qFe+qPxgW7dO/qY4b9mrn+315vX09OPJ05kO5aZ/W8Lxe/STrDQMD+aBn53sEfIM/biOg9vC/sEf6udp9ZE/prBv9WT/Uu5yG1GWtAc3zeDIhgMPH04vgcCtmXxZPo2xa2lyb0naT7//2qdi9qq0tMWYGOsZH7Kv1z/n+kNb6V99M74k7/OcDHw2v7go2EtjR96/L5ce6G6cWT9d02b7izbin5PGHfHemju3fXBxPf57MjlAXLzfVfiSMp2O4Lp7+6mXx9HevigUzTlfVbq8vjYsNu1eXN6wf2zfF53y3asSInSS/YJK5v7mo1hfN0/8gEFFXIKLuWEP37PFsl+OnEj54Ko3W/tRJW33oa2aBi/pUYdueTIvbPd5VfJ9GVdsWPjQGm42b9rmy0chenQO7Y/vmVfrzqng7WlsJL1ZuurxaFU6aJJkTrqMrJ1utl85VvryK3WgZep6PSPgJmgcC/rJYLnOpegcG6n06V/PvTmdPXRjJrn/+q3/+q3f+q0cpNG9LS5+7G9+Lc7cJVuY/ozCHEmXmyjrW5iGoTspzL68SiR7u1As1P38n1fVB9kV6Ve6XFXsQF6c7nG8uGPiX63zJZO2/WbDDR1Ps9/Vrv5vQ63anx1Drdjy3cr0sGHAHq7WTZKvICVerwMnzxHXSIPDSq6ul76dXgFx77gk5k16vf2NjYSsXbEPc5ieNW9HNuj6c/Zzyvz3x78n54ePdY0LD++b0RLx7HgMV77ahMh7+GWV8mq/PeRqS8NMOunw9j2jv/3XMa5HbX3wu9zfs1b44VBd71r9NccHedabm99Vxd7E7boqvIuLPtuUtsyQvjvurfLsyUvMHTX0jHP52WR2nIPy0z9Nodk1vh1sr6WGR5evQy5xVcJU6YRK4TrZ2V463jj2PbVtGcYIQeOuxzHpJ7843nwclORdhJ+06Me7U2e+eyzfEowYSYoeMQvP+d0rWD/f1MyxesxeRPbpNoz6LF9W+vpLFYAc2ntnruGw8yMXnYsNG8ZRV6O5wbxS6Teo2Yb3Jr/eX/aTEH4vmNWbXubnknfvbplieHkPQWZFg2opEfyYrMiT0CV/NmR97cJhtH46NeRMDhX9Zbpn6GOh628A3oubv892urCYhvd3rESi9Gwutprtx5gVpeOUsPY9peuj7Tpb5ubO88lZp5CdLLw8gr0rrOx74VQannE/W+Xl7/0jQdeRBz8736N0x3rgNn9iDdIz33cDE/afdrrk4dmMH+/y6Ldm7WE/m1m9zfsjvbnbsA3pS5Ad3vJf5wca5hN7963PBrtArfkQ8xjfDf5YX5lsV+PmRnT99zuyhF2Z5HDjhKs2c0M/XTN+vYidOgnWx9Hw3cFeAvidjp3l/unnc5t05uSinZ17x5OyvWLBncLZncLYnJeP8yCkN1+Dv/n71ytxvm5/AFdzxfyoCn5gn5Qg9r3P+L4X+Nhwq8+N3N/JaefbTIvRX/pXju1cRw+8oZPK88p3kaulGQbKMch/Db7ePAmsVenDK2SS6O284JuWo2+CON3Tu73ZDNm7CFe8QkfDdHWAdvl9WN/nnjxfcjk+p/OCeDwIo+41/Afij6LxdWX/gsB9K+OOFxfzlSf9PFP7T+OChLOt8Hbpu4MTR1coJg8J1Um+VOMGyWCarKEyyDIlGPMFs3Es+P9ls7pbujM1fp9j16Ay0kzPQTs72jM9+Oz+OmgzlrdpQdg1s53e0V3O+ZS4pP2G4qpb/xeyDwO+BLEMCT/rLB8Hff1H7N+lXmV/A+4HC0T3wVpHvLp3QXa+dcH0VOldr78pZh0t3Fa3XaZAgnhWvczX3rpX+jPMoeefbHnK3d47hIxe4d/7nyKc+4R7v9sZU/Icdey9WGxXnSn/Hhu6Vfutcah0iDpb/fvA2F2uuxSO5Jt0pfP8JqT7t9mcW6/rFO+52xfbwull8tRisRzttr6/ltEyrrN8fL0nDMEuzzPUSRmHX7KX6qdY/z/OzuIhz58rPPIft4jn5cuU6Xuxlae4v/cCtXRf1/q/zHfv/Oq6ovvxVuV4311e/25c3ZbE+5OXZy/m3bX61qe9FO36bjp5URLBijGlP+78P3RXXO+xPHwHsMTWr0OrB0Yaisv/8vrplb0rz8rI/Svb+37K/6p2aq65v7M0prul8xVodOVX7BOpb4HsnMX2Z39XXdZvfnV/o8iRt78vDzW9sUyMUi/A73/vOvfifxhheeH4Q/S87KTv2bfnvRgpuitOyPj8bRKavyuMt2429U2Vtz3zWk8/l6sAGQ1gPon3RKOjpSaWxn/pJ/QrnzV16w0Qq314zK1C/MEyWN6v3u7rL9d39crqEn9m1Nvf3Kt8X7wqGzWXd2VfLol2SJ1zp6jeLDU97/8CulY3A5qb/tGzuoGpr3CyEX9SP4ZYmUDym2dlXPwFv31M/hF+Gq34IX3mVqR/Cl16l6odE7SGJ+iFtjJIXqx/C46Ai9UN4bBXw4LNurZr6o3S71Wrqx/BQAOD5d3P/wADgMQjAAPBCHoCmfkjEI4nVD4m5L1X9kG4FsvohKYdN9UMyTqDqL6bLuVP9kC4URP0QpPk2zQBwRNjOWKofEfEZTwsSHWhItA9K9FDWVa8T1WgP12gf1+gA1+gQ1+gI1+gY1+gE1+gU1+hMQ6JdDYn2NCTaxyU6wCU6xCU6wiU6xiU6wSU6hSWaqzqiby4u0ZCcwxIdwBIdthKNinpoRdRDDVEPNETdB0V9aAhU7z+q6h6u6j6u6oFE1S8r9ilJG4K4cUl0ExU/lPubhXIjEa7zMa7zCa7zKa7zmYbMuxoy7+Ey7+MyH+AyH+IyH+EyH+Myn+Ayn8Iyzy0DIsQI3XmwzPuwzAewzIczyDwi2YGGZPugZA9lXvWJoZrt4Zrt45od2NDssNNs4UN/dqg2G9FjnzABoheaOCSW6DlxSCLRc+KQtNNzg5uVSQSeH8JndZ7fN/evnYH4QFoL0bCC2/JokwC35U/bCuW2AtqIELeY2x3EiES4EYlxI5LgRiSFjQhEhi5sRDzYiPiwEQlaI4KaHTte/UjD7IQaZifQMDsarnoH/WJA7Y6H2x2/szuwqRK91d8X+2LXrC6rtody22R5ckk7FZ7r99uborhbCAfOi1JsuX7c5fv9Jt+uHpzyXNYGmUcfKlvUGa2z7nxfFDtJb54sfmDtsxsk6NbrDfv33qRPcWcVrfaAsL2e3bOompTBwvjhjX+eb9k/e/G9//txe70RqohhZzJuwoWnpe/BpE2+OUtu8c9BBpVfmkligiXS+qZU+aoOy6in2Plhj9CDPgnNA7yy1ieCVGQ6hJKKSKDgtkKaRyjqiXAeiXEeSXAeQUxJBvOIC/OIB/OID/NI0PIITh0IQYQaBBFoEITGTJKDfsCiCOHhCOHjCAHYzqDDi5mAIRQDA7vY2lzyLc9u68C7xsjq2Naf84/F/oFSKYGMaORaufB4Hl4gqET0YtghDtHrg7YMGtUMter2+vOm6U95e7erPhWnzKuLOgD1Ylk0gaEkgiRfobNSIontMRHBHyJFMhxzHEdGXzvvb9ibY6PnIY4bEY4bMY4byAxFiuJGCyj4bC+CGx6MGz6KGy2gAFMzPPuXT3wbajpZCE3XnClAoEYjhNHRmEl1ZO4REoQQd6eEasw9An5HNUoyYsXeBwTo5HU6R2snPbdM3W8XfZkf1txtuSkP+e7+tJr6Dx5p/rI70D8tqqmF6rbd+LK4rXZ8URHBcCcBoX/80K+C3x0P9fqAql0iwY3QWVWb6uHGh+YslHCa+UCJFCANcqUREDjy2vy0q7YC+vUNzjTI7mhz9E3iYarE9cHXYkgKy6DHPTexuQNiGwyST1WzVHYeqiSoMdJQzKbgVb7lalMrS6ImLD2rCoSFY2Y4FJa3TXbcnUBa+ptcPbhZqjdlLJz1pWRzXUqf3QK6FLJpweCZjc+DefkcmN/mfI7gMB5lD3kOUdhu8RwPsUdg24Nh29eGbTt4HHV4bBG2IXDWCCx3ZPEsU7Cteq9Jcp6AbdF4MXch+mLw/Wm7LLZ7e+RL4PZs/rtwgIYqRGUPRkXfgyhxioY5ypKiYW/Hp5hguG1ntlL0dpmaQ1dITW/LzSfpfLjB7Pw0woVqRP4Is9hP5ON9kO8XuVNKijWNML7uJ9lEAJdaoB3pbYTdjJCXEUaTFEWTFmYQNHFhNPFQNGlhhtAdECWSDnNwNLIDM5EGzGisZHY0FlQ4fZSUQcCpI/MPio6ZkW+8Ad/YliklsJLFNc0SUNWXPZ2ZrYBZSzErGWFOTGMOikqj8B/w61sMSSJ5M6eErCeWORyeg8ykWGiY+ajl/DP6DtGIlLSgGV3uJVWniLy5oGcLwXWQixXEcevyUyHR5BFMIDFMIAlMIClKIC2zIATiwgTitQSCcwYyPxhrkEGkQQYaC+idft0OThPAPCGFEoqTgQph2RZxgGAN36ivGowx1wRY0Fl6Iy+KmB+A0CQxPqRWyEqMFXZ8HGLcAFwZYrYYOSzo8P3Z4tkzKSdM+iGsYCiHFXNisEcvItNoj0dEplqLRyy1E8IAEcEAEcMAkcAAkaIA0SIHAhAuChAtciBmPdVAjkQDOeIOOWy6QxAYCTVgpF/NZWXNsNOHKRnDjWh4q5IMIDkErox8nm8PdfpQA2rwB9Rg3YZ2OeDmIZ5wGi3Mw76jacbQs55ilolnZBmRaCr3LzWjCLWOSlLgqTXAOePRHXmDEoszrA+cMzJRNYBIGJI4Vywi5YlFFzyq+HpkTIQTFh5QM2JicBorhgkqgQkqRQmqZS6EoFxdgoJoKLFKQ3FHQzbZCokD6nOrqL4RfS7yh0PYka1R02jOlyCTku5JWyc8OL/u8m2bVHo0wxyC0qxKdSMmM3B3DUBvFiCTZHxE/Uy61IXSHfFu0U9N+QRRT1yDp/eqVI/hsNAJSd5KiO2wVaVitgP9rBDSzbOooMuHpzkYX9ZPt/5PpA/bvMm1/77arhqQYbd5u7p4WexzgSEUwYWF66ZzeOpzEEIudH5POeuEFlgHz/GKkEuKkkvLOkgittOoRbgl02CdtGMdnI8Q/44spT7lj5Hl8dQAjVDCLThojBeV9D8Sy0ocYkUaD8w9X1nSN/d1OjsMdhH3a45PWCWUg7+eVRFtFprqEqvO496SJGGl2zj3DTzbNlViLppqPw9fw6BHLLOVbGJ0w8JqxHQWKfYNmjDU9wInUlKywINdHtn5UAycgFZ1vM0DOjyL7WO783guXNAL9e64Xe1ys7fS41l1z1Hp2NTWxQANjzDCo6LxJV54BgY8RZQOXOk4nzSijCAHj6ywhQYoqSTPtQobZ0mJFCFINeJAuWtwqhknmGSkR+i2HN9GXqc2sAQAOlrFlKIglIBuvBqfXKmmYGWMPVddCuP5PGsiOYDmSscr0zWW9olhzGjyr0t/bCdySUxlI4OH+nEt5fcCyQsFu9FbS4WvKzdLfHy8zHf7G4PgSLfDrlmitcSU5891upt8/7w6HKrbN3UFyjflp77gq5j7wEX3YoBDknP2+akNFrTBOBbDOJbAOJaiOMZzfEIprF0NhMskCEf5rVIJwlHHyOpHUsfIitYQ3xgaFDiOxoSDiWZBxbEM8CW+3xiHBWIOO93EOT1+FBASPi7Yb0ewnD56ceiyGbfmmfKWJLMzBGxGXBX2XDVPjLiY4/TjqyQZoQdzaRJ7O/iJtsKSJNLGFt7+vLX6uens1ebg4pJzeFNt/1ysNZ6RR6fJpiAHTpPtwWmyW5DCS4rhFYQRCEpgCEppCCIXcrsd0NgJms40YCeVwM7bbfV50trIyrKak088IB+7KqCBXyYJdc5OIEqmFqqFdTrn2X0oJvOGYZ4vc9bcBZ1UbchOlWDb3a66reqz7OvTDP9iTyD/mNdNX3yfb5vwOQkoasdWKMXAmbqucILTzv+t4oqbN15LMI1oUKgGpcCx00ycYCaez+1nBIFdIm5zkBRZYzLGaRzHdMkOqS5WxcXr6sBafyjukvzccJpWeUfPh/dlflN9bD/TSGgZI6SBtuqcj2cK108IZeiO1+k0T1Yu7s+mqlYXd6fXQMPXNnXyFlfmm0r3eJ508ARyfEWSAfIk6iZevRDG0wjG0xjG04TjqYXAd866WC5LkoK9AQUbZEVyO/hVaUYJbTNTNk47NtZMqAYRuMiY2CPwOVZoKVH3bJMaGvGKos8788nXYE6c9KU4acvZmBg/JHVu1c5WeZnfFRfv2BcOncoMTP5kfNpBiuo5P29lXRjkrX7MK+/yWc8zfdvlvoaah5B20sdnkqZM7YTjWRB+FDEHkvTQO5jt6A+adwj2F4T1Ou2o116vUUQXgrjxsmc4PoMCVziFuQenMG/hGAHbEAbbCAbbGJ585ixsGV8Rp2sHq8gUc9bBpCGNApxIoaYdT+iTxT+Ou3J5I8NO3ZVi5lQmyzlu3rokT5dayI/TrevUZjdJDJyynXU6l94MdPIoRPbQGvEfqaI0skngcxPVtwTJvNaNg69jkGNc3OtZeVN7gakCVI4+GZ9tjrflVhiamM6SiZ4Gh3lGs4InEWSuuCdFFfeoPTDTvmLV0AX6RsBR54kmmNp0GRt/E/A09oIASuC6ppzPdj97hDQ+0mMqYgkCdOJ1xX3RcFp+z4dxO4BxO4RxO8JxO+a4DRO6ncqknq9D6H2WfSteaSA6guJ+tfVlio2mEpBX8hnT5J505K6p7tMfBdprF+mmJVnzFXk8pHncPEupE9CsDlLTROkZpwu0NI5xtEGLTueYfXzW7yh/TjqWdQkvuf5ISK/7reGPgZLZ9y351TTr94diYk7FrNt2PL1CH7auBwQmaZ0vBGIJkDEPf7UPn6THYcE1fI0YlTHXlpuP4ntpmPBDjN5T1K2cLn1yvFLo7JLoLIdttXgCOX4jMB3AMB3iMB3hMB1zmAawVVZbYhKZlR3UUJaFnqUN/NdZh71K4m9IwqmEhMGmxyrW/kKtdUo6UD7Xse4wiONAtAb72pXlnPRAtA3Nxf7aRT3lPB3S5G6lVIMjXonestVjfS4QnhJjPFHtRueU/zqfCd8ecXsD4n5EuvaFdE3nBvBnCUvSn5mny27Y4vmR7RHnL02sZAlQ7VjcQ6/2F5HquegKIgRY0AVEiAPo+iFyxER8rXD9kBapEDwMcDwMcTyMaDycIEqgbozTlemDiFJWBWQSQ0XdU4qT6EBzBudo9tWQMB0i4cRqfVugRXYRT/CUDChRpaeqam2KremYulSmSG31egqKBrVQFMLPH7kQqBMKuVUyWR9Y4XO71TgybNnmSf7oHz/MUJvD0cp8gDxJ3xp8K2I+vAgJ/wZQd04IHNtUAiIbhN0VrpkjHN3SV5zQ5T76JmAwe2Ujba3R8kZe0UVT4H4sthUN1NpF16lmeSUYSKnFaSKpU9ClY6S0jbCzC7OzB7Ozj7NzgLNziLNzRLMzSbR9lRejWARZdRc1MPbmA+MOukcq8aLa7Yu9Ma4Irw+FdtFoehRoH/lGRSmrZkd2qaM5FZm/2UidQHE7ZYUR7hlUpHncE3c5QudbdafhgYZXblulT+2iVTB9EpREX6dkWZkk8h0gvLGV0T7S70jNyiAScqB52lCQEM3xDpqpjCzB31iKTz9IwzFGgiD7lj5X51Pbmi+QXo95qR9lc6/SXx3YBfKJwqWDWqxB0NXD0dXH0TXA0TWE3b6cdoEUco4nqw006V6GuNqX4Oykexnw4JKsawEoCVbVziUwGc2AlbcBg4Z1qzYrLnjrajZrtzCocjOHvXLECUZnS+TtBDQ7WYUzHzFMtln62S6/kpc90Bx2JNf1ZyRXsRhfC5BrVAfqjDOc8oo8+iOLMtt9vZ0ZvnSok9IFemZkpEhEdaKv63n5DZ6ui3u2M4c2DaoUxX7OB5DjOUD2iqpHmsLlj3itBwQQPRwQfRwQAxwQQwNA1PKh+oKRoYGMhJiSKXtVyZCow9OEVRvzpmiQ2cHNVNDdWWaSOIWOXnD6s1H7RAnNiuo4F/e4aIZzimvQuro/2i1IEpNieSkUQ6wVEIpASpMYPQkRWmlVbYkHDWWGE9loRzpG1E7KIMTSKTYECc5o6pSX+Hm8M/ISQV8BGFXBa8aOxD2HWacuuqiRNIQ0UzL9UkyDoMvFocvDocvHoSuAoYtzGpBhzPEiCadNwh3kk+tL++gzki8mL0nkujHlZXNAnui1UIK858XmepevCgnmEe45wyxcshOnWjOsxqdNtNx8xGnVXSZOV+9a+MEvuvUTeVfMu9SVILKZdJygTn92qh8s/gYerImfT8CvdhBzZI3bX6hAjd4HOLTH3UGzTNtrdfb+YUYlWS/JJTHWQNm0OJbXsa7doa34ReN34Ds4/fN8y/7Zow9e7QuQLgJlgae1kz9TzUY9qOpkaMfIlVeiMgBRuDSVpDKVnHURcHVxcPVwcPVxcA0MwBViUFmRo0lwhcqG+UJ6JLzOasFmEhTisEokkDdc4Ds8M2BCOewaLarWO3XWE6mCewsmUaAnqRRSTclQr8ogJ2cqs/VEp0Q1Kac8V05XjlzaHsG1RqsHuroQ5m6zB6A6z+ymErCqJc6cAM4+knAicrltZ+5ronqPL0UcFD9XuQx70/oYRY+Xc5/yBqlytXlWt8dja/3pDrpEFIK6I1sAVp+nC0YRB8D1ouByUQmMjCmMjBlOjC5OjB5OjD5MjBwygfSujhdLIHOSTIH73JMpBJmSmkRTYKqfR0fEQZPmnwOqcjlLc2lT6+hYrrujSB+NK2TLy/xw2IytZteYTeZUZL7JC8MSe2WDSXLRFdrEWxsDLjVCXFu+aWuASOA0Pn9PEPY88/TgTK4pjbcwOzMN6RD2OKWqSCyssynadVHB61gxc8IwMkH8WzDMm0CXue76VI1vC9nu4yP+JPUSvPuNubuFIatmlD6yiuK8WYIUIQSu0OWspJwOYDdczAqvZZXA2J3C2J3h1O3i1O3h1O0bUDcE0LJKUFPUDQG0pKSTeJ5B6rjVKu8kASQYZ8XV3i+ru7sHGaes4izdUQ1njztBp/N2WHxnM2GYgPpqTzlazxZVkdKOWnU8TXo8Nec/gn0x0HvAvXO7bsVeaGuLtjEoFqf0t07gPGLUesOwC9EIqWioIxb5aHxhUlkuRRRFzBoSTdA1iaSQBURq8opEBos08BJFeIWiBKaoFKaoDIcoF4coD4Yozl1EMCBanUlWLmgS4yAik5T4maI4YAR3FCd7TXCKM6mvaYXiQPecvZBQbTfquHSPTWQjiGw050aEEcVzLP1JIbcdzFt2Vg8/4C1Jq3pFx8VOyHEhy5OTzHzm304opxis5slxOUVYwtDjmaDLVs2AEUkhmb9JoEHSJemXYA463IGetBSeEKOIl5ORVJORsxJAPilMPhkOPi4OPt4s4AMxTKzBMJLyJlPcA6B+xz2y1w/nHqOkxT0p/JZvNsXhIBajQUFvG3IEXKRLe0QULWhmI4wt1bLxAEPMIPS9u0Y/250TaVhxMQmA07FW5iNtDVgxCWBvntjC66+evzeIyVdMktjabhsLIxBagK29lURMXTERcMgQyi07laQIiUZreH0SvDxJAgNHCgNHhvOGa5E3OLwQYfwovKQ68JJowEusAS+RBrwMilEIbg+Z12UKhkzwRcospiUXQDYSBxPPSEsT3XKls0CmK9i6jN9zARHxFhp6UxKRr6OpEz8nfpnRU6RDTyic2QkWE0OYGSYJQ1ssuKf6ErjzudcIz4dyJ+mqDubFhXwp7tjNhg5Fu3ZVJtSeuD5+yYAJxy8ApmIYphIYplIYpjLOUjB+QeiRaiBOooE4MY04U1gExUuFNMaIh6bUag/qBExP7yrxwvB044mi7jdyosgXwo0ov2bX1lfpJ1HLdXCg+XwS0d/JkgiuFiqpcpi6cRD7tObpVkKCD0ROZoFQUYc2Z+E3wvqD4nKytvBqjnmcHrDMMjD6kyBkdAN6J5L15u8fxMGoe8DUxrEnhaRX+b4UxhdZ8Uf5PQfN8BU5Y8HHVG3Zwll9BfK3DzOUe5RUdtCnSVmuXZwmATaMYTZMYDZMYTbMYDTkNAmRoaSYwxSBQjSZaNCkpJbCFIHKHiZAk12tBB02xSEsEMKieBlK+FXoUSO63Z8gS9v4qM2+48xl3wb76uNpNiOepi2eKkWCQb5FM2KNO2KdhcUiPRhFadegekygA6PQpK1I9hVHDeyOG8Gq8G0AnYEUD85avrCrdGBjJlgDirqyCSMvRL4p9lZdggB2RTB2xTB2JTB2pTB2ZfrUpeP3gwhKUmFgirp00tnJnj5OXeofBFaoCwCUiV6YE4k/IBJpjRl7nXxIRbqzJqhjb5zAy1rpCqSIWNYBjbndRM6bdpOTasss07ni54SuPvNZwlhGTHi3sbQvkSlNhR1NKQSsq98UvmbtkSrwSYPCbFCr3MtnFCqhQWxIbJ7Qh2X+WksqJujTm8wLIaYxwIZHMFvFMFslMFulMFu1Ok4MRTAGzXc1OC3T4LRUg9MSDU6LpzmNYjs9GqPYTj10QJ0musT79llI1F0N6LA6TeB7qtmNeGJ+wTzB4Mc5JgrOz20dnlqANFkip0KF85WuTe26q85P/ff8jr1PgrMmdplMjHwmq4nJnvNM+pqjiWw37IDsMZ9DvzTPDAPRqWqhG02SZU9S2lOR4ERv6OCE6i/Qtr4b+eZ9tV016tM0cvGWqVQpmBCjKQxZTYGAl6QWgBzVRvGZWMGGCEaxGEaxBEaxdA4Ug7Aq08CqVAOrEg2skqSUn0IxKJ+vJH+8RqymvCQu1JBPkoFWc3R1STumumUNIoycrBFErd7IWstv25Kldm07RBSzmPy+yiPkWjC7i5EOD9hAEItOGghCpGHssjh+sm4ZeULfiCYGPDKefZ6l5DGfNrM+hS5JGy9NjYU4dfCk8SFMEhFMEjFMEolFkmiphPApolTi4VTCSQaikkyDSlINKkk0qCTWoBJJ4m6aZGyhAp1q2wBkfCWAN0AbGXgRsKJ+x0i1zlqemIdTRl89SMlCS8xi35fAC/CZtjv2A+bL01kap9+76oJZbs+tG6P8g4ruwWEOafK3DwrJz0897z2Dp78FTsFIhjqzMWooBR3z3AhThEVYHOOFxIPYIkHzZIg9xloi8VGkpzmWj0hzbXmRkhQTdgZP+R3AzBTCzBTBzBRzZkIpyy4ZIZTDyQiinEyDciRZoqfICKIcSa5nmoxw5w7UPk+pDBxCJ0YmD6FTHoP80ZKM+urQCZJ5UEiNCIE8U/3yE7PF+XZy4Y6ixU0H5EH+1lvct0UdhQNZXAMkGwW7aIQAQfxlzkli/hIpJdblqUnhqOeJwf36dZdvS2Hklv/Y3BH23KEQpK/IA5KlX0QLWMiWJHJGMdqpc6zYjLUeQw0wwdOlv8bS/sjxBICNAIaNEIaNCIaN2CJstOBCEDAKLpJs2FOwg9huDjsQuGQa4JJqgEuiAS4ay/Q0akxClxHg4OKT4DLBOhYIpWUdtalgLfpRTxg/QQmpDkbpU8I4RgJbaKZj3WOhddfJdDULdZgZdcT7KMn1Y6H1wNjc+0JzDxZIdgaeA8RIU3iApGCEDboPG/QANughbNAjiwY9nsOgQ8bZw40zN+iQuZHkVp5MjAkZdNxtoVG/TiMwBbpbAW6c6UTDE/ZcPcf4lD1P5rPn6mX8Jux59rj2fGxXjQy6kiH31OyDhRN22YRNypNqWGhbgVn4ahNHEpkwYXBVDSVoIj3YRPqwiQxgExlyE4kaVTgztY/Iqt8lErZpbomgds1pBOh6XA2jmmkYVY0EOBolyDTiKiCjGuJGlU5LO2GHIR8BXTmHPMSdtIpiO4qU0E25BVM/RFJD2Z6Fe1AlE5hnhSehH2+VkwuFMbRWkP5xllVOkgy29uy7rfAnfImCg8+JO+BnLWitPdha+7C1DmBrHcLWOtK31pbs6yA/rT1rjZgsbq0hy6iRrg77XMYtL51vdcJYQ5Y3xC0vneJ0wlhDltcjLe+EsUYeZUabXuqQlDa9OuKZCG0t+blsz7oLE0jbuaYuYybm+7Rn2YR1qdWtkyTKfcKgqXlmJkyc6pgHDZYHGywfNlgBbLBC2GC1Jg7ypErSbk6ZRciU+hrmysPNFfIQ+Zco9iGKmys62eKEhdPIHaER0gaZqwA3V3QqwwkLB5GHi9urbNpeESZuvgUIkvKqOs1JqncYmC/keybELUZgw2LgM3cO+KUDmgwPNhk+bDIC2GSE+iZD50sKkn9fQ/4xW4HLf4bLf4rLf4LLf4zLP52ybcJiQPIf4PLv4/Lv4fLPTQby/DOu//BXEbKkDC9g6UgS61NnkaQhmxBz5PnjIZEOPj/kgAQPCrEHC7EPC3EAC3Er3eyRf6g/J487ZrPrrn4qdvt2r+WOfd4Vq/fl4ab+c3u8vapdrJ4fRPUH6O+10zj8zve+c2tXK/+Z9eH/AWGt+TJdyQEA
```
</details>


### Steps to Reproduce

1. Go to '...'
2. Click on '...'
3. Scroll down to '...'
4. See error


### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

Android

### Additional Information

_No response_

--- a/android/src/com/unciv/app/AndroidLauncher.kt
+++ b/android/src/com/unciv/app/AndroidLauncher.kt
@@ -1,6 +1,7 @@
 package com.unciv.app
 
 import android.content.Intent
+import android.media.MediaScannerConnection
 import android.os.Bundle
 import android.view.View
 import androidx.core.app.NotificationManagerCompat
@@ -10,6 +11,7 @@ import androidx.work.WorkManager
 import com.badlogic.gdx.backends.android.AndroidApplication
 import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration
 import com.unciv.logic.IdChecker
+import com.unciv.logic.files.SAVE_FILES_FOLDER
 import com.unciv.logic.files.UncivFiles
 import com.unciv.ui.components.fonts.Fonts
 import com.unciv.ui.screens.multiplayerscreens.AddFriendScreen
@@ -56,6 +58,7 @@ open class AndroidLauncher : AndroidApplication() {
         MultiplayerTurnCheckWorker.createNotificationChannels(applicationContext)
 
         CoroutineScope(Dispatchers.IO).launch {
+            createSaveFolder()
             copyMods()
         }
 
@@ -107,6 +110,15 @@ open class AndroidLauncher : AndroidApplication() {
             if (!externalModsDir.exists()) externalModsDir.mkdirs() // this can fail sometimes, which is why we check if it exists again in the next line
             if (externalModsDir.exists()) externalModsDir.copyRecursively(internalModsDir, true)
         } catch (ex: Exception) {}
+    }
+
+    // Blind attempt to prevent issues like #9604, #9847, #10302, #13113, #15029, #15219
+    private fun createSaveFolder() {
+        val parent = getExternalFilesDir(null) ?: filesDir
+        val newFolder = File(parent, SAVE_FILES_FOLDER)
+        if (!newFolder.mkdirs()) return
+        // Force MediaStore update
+        MediaScannerConnection.scanFile(context, arrayOf(newFolder.absolutePath), null, null)
     }
 
     override fun onPause() {

--- a/core/src/com/unciv/logic/files/UncivFiles.kt
+++ b/core/src/com/unciv/logic/files/UncivFiles.kt
@@ -29,7 +29,7 @@ import com.unciv.utils.debug
 import kotlinx.coroutines.Job
 import java.io.Writer
 
-private const val SAVE_FILES_FOLDER = "SaveFiles"
+const val SAVE_FILES_FOLDER = "SaveFiles"
 private const val MULTIPLAYER_FILES_FOLDER = "MultiplayerGames"
 private const val AUTOSAVE_FILE_NAME = "Autosave"
 const val SETTINGS_FILE_NAME = "GameSettings.json"

